viral/ensemble_reactivation.py

```
@@ -0,0 +1,515 @@
+    if verbose:
+        print(f"# frames with running: {running_bouts.shape[1]}")
+        print(f"# place cells: {running_bouts.shape[0]}")
+        print(f"# significant components (Marcenko-Pastur): {ensemble_matrix.shape[1]}")
+    # OFFLINE
+    reactivation_strength = offline_reactivation(
+        reactivation=reactivation, ensemble_matrix=ensemble_matrix
+    )
+    preactivation_strength = offline_reactivation(
+        reactivation=preactivation, ensemble_matrix=ensemble_matrix
+    )
+    pcc_scores = get_normalised_pcc_scores(
```

Ruff (0.8.2)

Local variable `pcc_scores` is assigned to but never used

Remove assignment to unused variable `pcc_scores`

(F841)